modules/gdscript/gdscript_parser.cpp

```diff
@@ -4038,7 +4038,7 @@ void GDScriptParser::_parse_class(ClassNode *p_class) {
 
 				if (!_enter_indent_block(block)) {
 
-					_set_error("Indented block expected.");
+					_set_error(vformat("Indented block expected after declaration of \"%s\" function. ", function->name));
```

Comment:
Is `vformat` the best way to do this?

Reply:
Yes, it's a string formatting method which avoids having to concatenate strings together. It's also more localization-friendly (although we intentionally don't translate those error messages to make them easy to look up online).

Reply:
TY